src/stars-style.css

```
@@ -0,0 +1,48 @@
+.stars-rate_container {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 4px;
+}
+
+.stars__star {
+  height: 12px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.stars {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding-left: 2px;
+  padding-right: 2px;
+  padding-bottom: 2px;
+  height: 16px;
+}
+
+.stars--0 > .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 > .stars__star:nth-child(1n+2) {
+  background-image: url(./images/star.svg);
+}
```

```suggestion
.stars--0 > .stars__star,
.stars--1 > .stars__star:nth-child(1n+2) {
  background-image: url(./images/star.svg);
}
```
згрупуй селектори через кому, щоб не повторюватись